assets/queries/azureResourceManager/account_admins_not_notified_by_email/query.rego

```diff
@@ -0,0 +1,41 @@
+package Cx
+
+import data.generic.common as commonLib
+
+CxPolicy[result] {
+	types := ["Microsoft.Sql/servers/databases/securityAlertPolicies", "securityAlertPolicies"]
+	doc := input.document[i]
+
+	[path, value] := walk(doc)
+	value.type == types[_]
+
+	properties := value.properties
+	properties.emailAccountAdmins == false
```

Comment:
Maybe we should verify, before line 13, if `properties.state == "Enabled"`. The same could be verified in the other rule